dictionary/index.js

```
@@ -0,0 +1,39 @@
+/**
+ * Dictionary module providing localized messages and prompts
+ * @module dictionary
+ * @property {Object} wallet - Wallet-related messages and prompts
+ * @property {Object} common - Common messages shared across the application
+ * @function addLocale - Add a new locale to the dictionary
+ * @example
+ * const { wallet, common } = require('./dictionary');
+ * console.log(wallet.SIGN_MESSAGE_PROMPT);
+ * console.log(common.INVALID_ADDRESS);
+ */
+const fs = require('fs');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)